### PR TITLE
Add a tiny example for an explanation of an anonymous function

### PR DESCRIPTION
**Pull request by komainu8**
Dear, developers.

This is trivial modify.

We added a tiny example into "Tour of Scala - Basics" page.
Because in my opinion a little difficult to understand how to give a value to the anonymous function.

### CODE REVIEW

File: _tour/basics.md [hunk 1]
```diff
@@ -110,6 +110,14 @@ You can define an anonymous function (i.e. no name) that returns a given integer
 
 On the left of `=>` is a list of parameters. On the right is an expression involving the parameters.
 
+You can give value to the anonymous function as below.
+
+{% scalafiddle %}
+```tut
+println(((x: Int) => x + 1)(5)) //6
```
Comment (exoego) [inline _tour/basics.md]: I do not think it is not a common practice to pass arguments to anonymous function diretly.
I suggest a below.

```scala
val func = (x: Int) => x + 1
println(func(5)) // 6
println(func(10)) // 11
```

Comment (kotobotov) [reply _tour/basics.md]: i think it's good to have both example

Comment (komainu8) [reply _tour/basics.md]: Thank you very much for your comments.
I understood my example is strange.

The example that @exoego suggested has been already exist. Therefore, I just close this PR.

Comment (komainu8) [reply _tour/basics.md]: Thank you very much for your comments.
I understood my example is strange.

The example that @exoego suggested has already existed. Therefore, I just close this PR.

**Pull request closed without merging**